src/gsi/constants.f90

```diff
@@ -31,6 +31,7 @@ module constants
 !   2016-02-15 Johnson, Y. Wang, X. Wang - define additional constant values for
 !                                          radar DA, POC: xuguang.wang@ou.edu
 !   2019-09-25  X.Su     - put stndrd_atmos_ps constant values
+!   2022-03-01  X.Lu & X.Wang  - increased max_varname_length for HAFS dual ens. POC: xuguang.wang@ou.edu
```

Comment:
where is the corresponding change? From this PR review interface, there is no changes in this file except for this comment line.